sycl/test/basic_tests/event.cpp

```diff
@@ -72,4 +72,38 @@ int main() {
     assert(hash == std::hash<cl::sycl::event>()(WillEventCopy));
     assert(Event == WillEventCopy);
   }
+
+  // Check wait and wait_and_throw methods
```

Comment:
```suggestion
  // Check wait and wait_and_throw methods do not crash
```
Am I right?
Otherwise it's not clear how do check the behavior of these methods.

Reply:
You are right, fixed.